include/sound/compress_driver.h

```diff
@@ -34,11 +34,14 @@ struct snd_compr_ops;
  * @total_bytes_transferred: cumulative bytes transferred by offload DSP
  * @sleep: poll sleep
  * @private_data: driver private data pointer
+ * @dma_area: DMA area
+ * @dma_addr: physical bus address (not accessible from main CPU)
```

Comment:
surely the ring buffer can be access by the CPU to read/write stuff?

Reply:
I've used the exact same description as in pcm.h (snd_pcm_runtime struct).

Reply:
two wrongs does not make it right. this address is accessible from the main CPU: yes/no?

Reply:
Not an expert here, but this is probably architecture dependent. On x64 platforms, CPU should be able to access these, yet on different ones that might not be the case: address seen from DMA side may differ from one seen by CPU.